File: Linear_Search.cpp
//This code returns the index at which element to be searched is present.


#include<iostream>
#include<bits/stdc++.h>
using namespace std;

int linearsearch(arr[],size,element)
 {
  for(int i=0,i<size;i++)
  {
  if(arr[i]==element)
  return i;
 
  }
 return -1;
 }
//Let n is the size of the array and k is the elment to be searched.
int main()
{
int n,k;
cin>>n>>k;
int a[n];
for(int i=0,i<n;i++)
{
cin>>a[i];
}
int ans = linearsearch(a,n,k);

cout<<ans;

return 0;
}

File: Algorithms/Linear-search/linear-search.js
function linearSearch(array, element) {
  for (var i=0; i<array.length; i++) {
    if (arr[i] == element) {
      return i;
    }
  } return null;
}

var array = [1,2,4,5,7,8,9,2];

linearSearch(array, 0); // returns null
linearSearch(array, 2); // returns 1